Backpack-SwiftUI/Card/Classes/BPKCard.swift

```diff
@@ -36,6 +36,8 @@ public struct BPKCard<Content: View>: View {
     private let padding: Padding
     private let cornerStyle: CornerStyle
     private var tapAction : () -> Void = {}
+    
+    @State private var selectionOverlayOpacity = 0.0
```

Comment:
Added this as an effect to be had when the user clicks on a card, although it's not the exact same feeling as when it was a button :(

(we cannot support highlighting if it's not a button, that's why we went with buttons back then!)